src/pages/docs/creators/ray/cluster-yaml.md

```diff
@@ -75,8 +75,12 @@ initialization_commands: []
 
 ### Provider section
 
-The whole "provider" section describes quite a lot of Golem node provider internals. 
-Some of these properties interact with how Ray on Golem works in general, so be careful with those that you're unfamiliar with as changing them may render your cluster unusable.
+The whole "provider" section describes various Golem node provider internal parameters. 
+
+Some of these properties influence how Ray on Golem works in general, but we still encourage you to experiment with various settings. 
```

Comment:
```suggestion
Some of these properties influence how Ray on Golem works in general, but we still encourage you to experiment with different settings. 
```

to avoid repetition of `various`